packages/server/src/Server.ts

```diff
@@ -113,18 +113,22 @@ export class Server extends EventEmitter {
   private openIntroductionConnection(socket: WebSocket, userName: UserName) {
     this.peers[userName] = socket
 
-    socket.on('message', () => {try {this.handleIntroductionRequest(userName)} catch(e) {return}})
```

Comment:
This wouldn't have worked - `this.handleIntroductionRequest(userName)` returns a _function_, which is the handler for the `message` event. So it would have had to be something like 

```js
socket.on('message', data => {
  try {
    this.handleIntroductionRequest(userName)(data)
  } 
  catch (e) {
    return
  }
})
```

Reply:
I went through and tested each of my possible revisions before suggesting them, and so while it certainly shouldn't have worked, given that case, it did. It stopped the server from crashing on receipt of bad data.

 However, I could never test to see if the server still functioned normally given my change, as I still have not quite figured out how to properly connect clients to one another. Anyways, thanks for fixing it!